go.sum

```diff
@@ -1,334 +1,4 @@
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210922222656-bd57995cd433/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210923225427-0bc3ac81ad3b h1:6YL0uXMXqm58wSuPFvmIXcIXToVU4R50uFbcp+4NYIA=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210923225427-0bc3ac81ad3b/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210924225253-57293c1bcd78 h1:k2vHpK53n+T6Montgzvxcx0JsINlSEbqDSf8yj6kw/4=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210924225253-57293c1bcd78/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210927165259-aff0a775fc40 h1:icVNbfNOXF8IztWEdECVQSzl/yw8xBnL52iQAdew55s=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210927165259-aff0a775fc40/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210928001316-cc7f287da3db h1:X5klpZ26Bh/WIdy2NIl8DZ0/Gu6cJM5FT2OOj8JwW+g=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210928001316-cc7f287da3db/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210928224621-1a31bb6345c4 h1:BGJTVpDVkNsN/OGgYAK1/y0Hi+s0rU8HA9tx1lYQ+iE=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210928224621-1a31bb6345c4/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210929161255-54a6cab9652e h1:3+sIFIqEBxhmCSobYemIJYZ2QxUKD/R5vvtJdC2Tv9c=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210929161255-54a6cab9652e/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210929203851-6745cbc14043 h1:xBCnuIvC2Vvw8921K5hu2KuPegJsNiR8vFuztZK8O2Q=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210929203851-6745cbc14043/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210929230319-3698f5fb7f93 h1:yYhEu11BHbsLBjQBnrYz+m21Y+8l37DecyyYxZsMytk=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210929230319-3698f5fb7f93/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210930153011-18492170be0c h1:6sttFbOOBPA1u16vXKaijOuyWZPQJWaJfJK2F8ohtZs=
-github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210930153011-18492170be0c/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
 github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210930200502-0d9a549fe51c h1:jLXLwxAxTA+BQX1m6YtO1Km6yRHTwIK36lkqwns4h2o=
 github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210930200502-0d9a549fe51c/go.mod h1:081/gGTOxNFBjrLQ3QvsyP34iiWgmmKDtoi5falfsuo=
+github.com/pulumi/pulumi-hugo/themes/default v0.0.0-20210930213125-8af5768fe82b h1:io91gbHa/SidSdmS/py3sVb6hi1vCKgycGEBT/rx6AQ=
```

Comment:
You might need to run a `hugo mod tidy` as well. I expect this file to have nothing but just the versions listed in `go.mod`. Or did you run that already and this is the result?

Reply:
Conferred with Praneet and we're going to try running `tidy` as well during the scheduled workflow, since that should always result in go.sum only containing what's in go.mod.